### ISSUE TITLE
Issue when fuzzy matching

### ISSUE THREAD

**Issue by ramya-rao-a** [emmetio/css-snippets-resolver#9]
@sergeche 

Consider the abbreviation `minh`.
Below are the snippet keys that have the highest score of 0.41666667 when fuzzy matching
- `mir`
- `mih`
- `miw`

Shouldnt `mih` get a higher score than the other 2 given the fact it is the only one that has `h` from the abbreviation?

**Comment by sergeche** [comment]
Yeah, seems like `mih` should have higher score, will tune scoring logic
